pydm/data_plugins/epics_plugins/psp_plugin_component.py

```diff
@@ -325,6 +365,38 @@ def send_ctrl_vars(self):
         if self.ctrl_hlim:
             self.upper_ctrl_limit_signal.emit(self.ctrl_hlim)
 
+        if self.alarm_hlim is None:
+            try:
+                self.alarm_hlim = self.pv.data['alarm_hlim']
+            except KeyError:
+                pass
+        if self.alarm_hlim:
+            self.upper_alarm_limit_signal.emit(self.alarm_hlim)
+
+        if self.alarm_llim is None:
+            try:
+                self.alarm_llim = self.pv.data['alarm_llim']
+            except KeyError:
+                pass
+        if self.alarm_llim:
```

Comment:
Hmm, should the check be `alarm_llim is not None`?
Or do we not expect a signal for when, e.g., `alarm_llim == 0.0` and `alarm_hlim == 10.0`?

Reply:
Yes, absolutely. Good catch, fixed here: fa4a306213866e0bfaec03cac4153c1bbbd29f82

Reply:
Yes, absolutely. Good catch, fixed here: fa4a306213866e0bfaec03cac4153c1bbbd29f82